### PR TITLE
docs(CONTRIBUTING.md): Clarify contribution process

### PR DESCRIPTION
**Pull request by jgiardino**
Clarify contribution process related to using the project board, labels, creating issues, and adding
less/sass files.

**What**: Updates CONTRIBUTING.md

**Link**: [https://github.com/jgiardino/patternfly-react/blob/contributing/CONTRIBUTING.md](https://github.com/jgiardino/patternfly-react/blob/contributing/CONTRIBUTING.md)

### CODE REVIEW

**Comment by jgiardino** [comment]
@priley86 - for issue #105, were there specific updates that you wanted to include in CONTRIBUTING.md as a result of the updates you made in #101?

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,44 +1,116 @@
...
-2. Create an `enhancement` issue to indicate you'd like to contribute this component to PatternFly React.
-  - At this time, the PatternFly React design team will review the enhancement issue and verify that the component passes the decision tree. Only components that pass will be accepted into the repo. If the component does not pass, the team will work with you on identifying any sub-components that can be extracted that would be needed by more than one project.
+### Creating Issues for New Components
+
+To create an issue for adding a new component to the repo, please observe the following process:
+
+1. If the component does not yet exist as a PatternFly component, check the [PatternFly Decision Tree](https://github.com/patternfly/patternfly-design/blob/master/resources/decision-tree/PatternflyDecisionTree.pdf) to see if your component meets the criteria for an addition to PatternFly. If you're unsure, reach out to us on the patternfly@redhat.com mailing list.
+2. If the component exists in PatternFly or passes the Decision Tree, then create an issue with the following details:
+  1. Assign the label `enhancement` to the issue
```
Comment (serenamarie125) [inline CONTRIBUTING.md]: @jgiardino formatting of this doesn't seem correct.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,44 +1,116 @@
...
-2. Create an `enhancement` issue to indicate you'd like to contribute this component to PatternFly React.
-  - At this time, the PatternFly React design team will review the enhancement issue and verify that the component passes the decision tree. Only components that pass will be accepted into the repo. If the component does not pass, the team will work with you on identifying any sub-components that can be extracted that would be needed by more than one project.
+### Creating Issues for New Components
+
+To create an issue for adding a new component to the repo, please observe the following process:
+
+1. If the component does not yet exist as a PatternFly component, check the [PatternFly Decision Tree](https://github.com/patternfly/patternfly-design/blob/master/resources/decision-tree/PatternflyDecisionTree.pdf) to see if your component meets the criteria for an addition to PatternFly. If you're unsure, reach out to us on the patternfly@redhat.com mailing list.
```
Comment (serenamarie125) [inline CONTRIBUTING.md]: @jgiardino trying to figure out what this means ... is this implementation of something which is not yet a documented PatternFly design pattern?

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,44 +1,116 @@
...
+2. If the component exists in PatternFly or passes the Decision Tree, then create an issue with the following details:
+  1. Assign the label `enhancement` to the issue
+  2. Include the text “Component -“ in the beginning of the title if the issue captures a new component
+  3. If the component currently exists in patternfly, include a link to it.
```
Comment (serenamarie125) [inline CONTRIBUTING.md]: Could we reword this to 

If the component is documented as a [PatternFly Pattern Library](http://www.patternfly.org/pattern-library/), include a link to it.

**Comment by priley86** [comment]
@jgiardino this looks good to me and agree w/ @serenamarie125 's comments. I made a few slight tweaks to the PR template in #101, but it looks like those are already covered in this last section of this contribution guide - "Please be sure to include all of the following in your PR:". The PR template now has similar language which hopefully helps new contributors deploy their storybook correctly.

File: CONTRIBUTING.md (hunk 2 above)
Comment (jgiardino) [reply CONTRIBUTING.md]: Yes — components that don't exist yet in patternfly core could be added to patternfly react assuming they pass the pf decision tree, so that these contributions aren't blocked.

File: CONTRIBUTING.md (hunk 3 above)
Comment (jgiardino) [reply CONTRIBUTING.md]: 👍

**Comment by jgiardino** [comment]
@jeff-phillips-18 I updated the section "Adding Styling for your Components." Can you confirm if these updates are accurate?

**Comment by jgiardino** [comment]
Hey @serenamarie125,
Will you take a look and let me know if you have any further suggestions?

**Comment by jeff-phillips-18** [comment]
@serenamarie125 Merged this PR, if you have any further issues/suggestions please open a new issue.
